package.json

```diff
@@ -30,9 +30,12 @@
 		"format": "prettier --write .",
 		"format:check": "prettier --check .",
 		"generate:configs": "ts-node tools/generate-configs",
-		"generate:rules-list": "ts-node tools/generate-rules-list",
-		"lint": "eslint . --max-warnings 0 --ext .js,.ts",
-		"lint:fix": "npm run lint -- --fix",
+		"generate:rules-list": "npm run build && eslint-doc-generator --path-rule-list \"../README.md\" --path-rule-doc \"../docs/rules/{name}.md\" dist/",
+		"lint": "npm-run-all \"lint:*\"",
```

Comment:
This will run both `lint:js` and `lint:js:fix`, right? Ideally we only need the former. Can we rework the scripts to avoid running both?

Reply:
`lint:*` will only run `lint:js` but not `lint:js:fix`. That's the convention of npm-run-all:

https://github.com/mysticatea/npm-run-all/blob/master/docs/npm-run-all.md#glob-like-pattern-matching-for-script-names

Reply:
Ooh I see. Great!